backend-service/src/main/java/com/smartparkinglot/backendservice/controller/ReservationController.java

```diff
@@ -45,9 +47,18 @@
         return new ResponseEntity<Reservation>(reservationService.addOrSave(reservation),HttpStatus.CREATED);
     }
 
-    @DeleteMapping
+    @DeleteMapping("/admin-delete")
+    @Operation(summary = "DONT USE it for deletion")
```

Comment:
[This Comment](https://github.com/bitirme-projesi-beu/backend-service/pull/14/files#r433806525) includes this one too.